components/ide/jetbrains/backend-plugin/gradle.properties

```diff
@@ -1,7 +1,7 @@
 pluginVersion=0.0.1
 gitpodVersion=dev
 # Supported environments: stable, latest (via https://github.com/stevesaliman/gradle-properties-plugin)
-environmentName=latest
+environmentName=stable-rider
```

Comment:
This file seems generic.

Should this value be something generic, too...like stable or latest?

Reply:
In other words, being specific to rider seems odd.

Reply:
> This file seems generic.

It's not, we don't use file unless we want to write code for backend-plugin, then we need to follow steps wrote in BUILD.yaml to build proper plugin (latest / stable / latest-rider /stable-rider)

README.md is kind of outdated because of the breaking changes of Intellij Platform Plugin 2.0. I don't plan to update them until all stable IDEs are 2024.2

Reply:
Got it, thank you, @mustard-mh ! 🫶

Reply:
Btw, `environmentName=xxx` will be rewrite when building